python/doc/examples/probabilistic_modeling/stochastic_processes/plot_create_and_manipulate_arma_process.py

```diff
@@ -28,17 +28,17 @@
 #    :math:`(b_1, \dots, b_q)` for the MA-coefficients
 #
 # -  a list of square matrix
-#    :math:`(\underline{\underline{A}}_{\, 1}, \dots, \underline{\underline{A}}{\, _p})` for the
+#    :math:`(\vect{A}_{\, 1}, \dots, \vect{A}{\, _p})` for the
```

Comment:
I think that this is a matrix so that the right macro is `\mat` (according to [math_notations](https://github.com/openturns/openturns/blob/b5fd8338850d1255233f64a0e0c27d963d6f7daf/python/doc/math_notations.sty#L61)):
```suggestion
#    :math:`(\mat{A}_{\, 1}, \dots, \mat{A}{\, _p})` for the
```